cf.sh

```diff
@@ -1,3 +1,9 @@
 #!/bin/bash
 echo "------ Starting APP ------"
-gunicorn foia_hub.wsgi:application --log-file -
+if [ $CF_INSTANCE_INDEX = "0" ]; then
+    echo "----- Migrating Database -----"
+    python manage.py migrate --noinput
+    echo "----- Loading Agency Contacts -----"
+    python manage.py load_agency_contacts
+fi
+waitress-serve --port=$VCAP_APP_PORT foia_hub.wsgi:application
```

Comment:
@khandelwal Actually, it should be gunicorn. I accidentally left it in after testing.